java/src/main/java/gov/doc/ntia/sigmf/ext/global/algorithm/TimeSeriesPower.java

```diff
@@ -0,0 +1,14 @@
+package gov.doc.ntia.sigmf.ext.global.algorithm;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.beans.Transient;
+
+public class TimeSeriesPower extends TraceDataProduct {
```

Comment:
Should add a `units` property, overriding the one inherited from `TraceDataProduct`, such that `units` is required instead of optional (matching the implementation for `PowerSpectralDensity`)